arches_controlled_lists/src/arches_controlled_lists/widgets/ReferenceSelectWidget/ReferenceSelectWidget.vue

```diff
@@ -33,8 +29,8 @@ const nodeData = ref();
 const widgetData = ref();
 
 onMounted(async () => {
-    widgetData.value = await fetchWidgetData(props.graphSlug, props.nodeAlias);
-    nodeData.value = await fetchNodeData(props.graphSlug, props.nodeAlias);
+    // widgetData.value = await fetchWidgetData(props.graphSlug, props.nodeAlias);
```

Comment:
We'll need to update the ReferenceSelectWidget to conform with new pattern in component lab - I volunteered to roll on that, just making a note here.